bc/core/models.py

```diff
@@ -18,3 +19,41 @@ class AbstractDateTimeModel(models.Model):
 
     class Meta:
         abstract = True
+
+
+class BannerConfig(models.Model):
+
+    is_active = models.BooleanField(
+        default=False,
+        help_text="If another config is currently active, enabling this one will deactivate the first one.",
+    )
+    banner_title = models.CharField(max_length=255, null=True, blank=True)
+    banner_text = models.TextField(null=True, blank=True)
+    banner_button_text = models.CharField(max_length=40, null=True, blank=True)
+    banner_button_link = models.URLField(null=True, blank=True)
```

Comment:
These should all remove the `banner_` namespace. So, `banner_title` should be `title`, etc. 

That makes the code:

```
my_banner.title
```

Instead of:

```
my_banner.banner_title
```